modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account/add_user.jsp

```diff
@@ -0,0 +1,90 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+AccountDisplay accountDisplay = (AccountDisplay)request.getAttribute(AccountWebKeys.ACCOUNT_DISPLAY);
+
+PortletURL backURL = renderResponse.createRenderURL();
+
+backURL.setParameter("mvcRenderCommandName", "/account_admin/edit_account");
+backURL.setParameter("screenNavigationCategoryKey", AccountScreenNavigationEntryConstants.CATEGORY_KEY_USERS);
+backURL.setParameter("backURL", String.valueOf(backURL));
+backURL.setParameter("accountEntryId", String.valueOf(accountDisplay.getAccountId()));
+
+portletDisplay.setShowBackIcon(true);
+portletDisplay.setURLBack(String.valueOf(backURL));
+
+renderResponse.setTitle((accountDisplay == null) ? "" : LanguageUtil.format(request, "add-new-user-to-x", accountDisplay.getName(), false));
+%>
+
+<portlet:actionURL name="/account_admin/account/add_user" var="addAccountUsersURL" />
+
+<liferay-frontend:edit-form
+	action="<%= addAccountUsersURL %>"
+>
+	<liferay-frontend:edit-form-body>
+		<h2 class="sheet-title">
+			<%= LanguageUtil.get(request, "information") %>
+		</h2>
+
+		<div class="sheet-section">
+			<h3 class="sheet-subtitle">
+				<%= LanguageUtil.get(request, "user-display-data") %>
+			</h3>
+
+			<aui:row>
+				<aui:col width="<%= 50 %>">
+					<aui:field-wrapper cssClass="form-group lfr-input-text-container">
+						<aui:input label="screen-name" name="name" required="<%= true %>" type="text" value="<%= null %>" />
```

Comment:
Does the `null` value need to be there?

Reply:
Sorry, I should have left a comment. This jsp is temporary. We are waiting on the backend to be finished so we can get the User object.

Reply:
Got it.  Thanks!